modules/test_user_dashboard/spec/services/update_user_spec.rb

```diff
@@ -25,7 +25,7 @@
       end
 
       it 'sets the test account to be checked out' do
-        expect(tud_account.checkout_time).to eq(@timestamp)
+        expect(tud_account.checkout_time.to_f.round(3)).to eq(@timestamp.to_f.round(3))
```

Comment:
Rails 7 introduced some issues with time stamp precision by removing defaults. This [investigation](https://www.toptal.com/ruby-on-rails/timestamp-truncation-rails-activerecord-tale) dives into it a bit. I attempted a few cleaner ways (such as [monkeypatching & rounding Time](https://stackoverflow.com/questions/55632622/how-to-limit-rails-datetime-resolution-to-microseconds)) but ultimately it was conflicting with Timecop and I went for the easier approach as it was only affecting a few files.